Add tutorial for creating automatically xlf translation file - Update new-system.md (#1720)
PrestaShop/docs: modules/creation/module-translation/new-system.md

Note that these files will only work with PrestaShop 1.7.8 and over. If you need to ensure compatibility with previous versions, read below.
{{% /notice %}}

### Recommanded practice for creating module translations

> For module developpers, if you want to incorpore your final translations into your own module without creating the xlf file by hand, the recommanded practice is to :

Reviewer: ```suggestion
For module developers, if you want to incorporate your final translations into your own module without creating the xlf file by hand, the recommended practice is to :
```

> For module developers, if you want to incorporate your final translations into your own module without creating the xlf file by hand, the recommended practice is to :

Reviewer 2: ```suggestion
For module developers, if you want to incorporate your final translations into your module without creating the xlf file by hand, the recommended practice is to:
```

> 1. Prepare your translations (see "Making your wordings appear in the translation interface")

Reviewer: ```suggestion
1. Prepare your translations (see "[Making your wordings appear in the translation interface]({{<relref "#making-your-wordings-appear-in-the-translation-interface">}})")
```

> 1. Prepare your translations (see "[Making your wordings appear in the translation interface]({{<relref "#making-your-wordings-appear-in-the-translation-interface">}})")

Reviewer 2: ```suggestion
1. Prepare your translations (see "[Making your wordings appear in the translation interface]({{<relref "#making-your-wordings-appear-in-the-translation-interface">}})").
```

> 2. Translate them manually in the back office

Reviewer 2: ```suggestion
2. Translate them manually in the back office.
```

> 3. Export the translations (see "Exporting translations") and add them in your module translations folder

Reviewer: ```suggestion
3. Export the translations (see "[Exporting translations]({{<relref "#exporting-translations">}})") and add them in your module translations folder
```

> 3. Export the translations (see "[Exporting translations]({{<relref "#exporting-translations">}})") and add them in your module translations folder

Reviewer 2: ```suggestion
3. Export the translations (see "[Exporting translations]({{<relref "#exporting-translations">}})") and add them to your module translations folder.
```

> These translations will be set up in Prestashop during the installation of the module.

Reviewer 2: ```suggestion
These translations will be set up in PrestaShop during the module's installation.
```
